### PR TITLE
Reword the Official Images description

### PR DESCRIPTION
**Pull request by wglambert**
Add short bullet points for the core ideas of the program, and a few sentences as a summary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,17 +37,21 @@
 
 ## What are "Official Images"?
 
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images hosted on Docker Hub. The main tenets are:
+
+- Support multiple architectures
```
Comment (tianon) [inline README.md:42]:
```suggestion
- Support [multiple architectures](#architectures-other-than-amd64)
```

File: README.md [hunk 2]
```diff
@@ -37,17 +37,21 @@
 
 ## What are "Official Images"?
 
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images hosted on Docker Hub. The main tenets are:
```
Comment (tianon) [inline README.md:40]:
```suggestion
The Docker Official Images are curated images [hosted on Docker Hub](https://hub.docker.com/u/library). The main tenets are:
```

File: README.md [hunk 3]
```diff
@@ -37,17 +37,21 @@
...
+
+- Support multiple architectures
 
-## What do you mean by "Official"?
+- Exemplify Dockerfile best practices
```
Comment (tianon) [inline README.md:44]:
```suggestion
- Exemplify [`Dockerfile` best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
```

File: README.md [hunk 4]
```diff
@@ -37,17 +37,21 @@
...
 
-In many cases, the images in this program are not only supported but also *maintained* directly by the relevant upstream projects.
+- Actively rebuilt for updates and security fixes
```
Comment (tianon) [inline README.md:46]:
```suggestion
- [Actively rebuilt](#library-definition-files) for updates and security fixes
```

(Hopefully this one will have a better place to link at some point, but for now this is the most "canonical" location.)

File: README.md [hunk 5]
```diff
@@ -37,17 +37,21 @@
 
 ## What are "Official Images"?
 
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images hosted on Docker Hub. The main tenets are:
+
```
Comment (tianon) [inline README.md:41]: Thoughts on adding one more at the top here?

```suggestion

- Focus on [Free](https://www.fsf.org/) and [Open-Source](https://opensource.org/) Software

```

File: README.md [hunk 6]
```diff
@@ -37,17 +37,21 @@
...
 
-For some, they're developed in collaboration with the upstream project (or with the explicit blessing of the upstream project).
+- Adhere to upstream recommendations
 
-In all cases, we strive to create images that are true to the upstream project's vision for how their project is intended to be consumed, occasionally adding additional behavior to make them more friendly to use within Docker / containerization in general.
+- Add minimal quality-of-life behavior for the container environment where fit
+
+See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
 
-The name of this program was chosen in an attempt to reflect that upstream-first focus (although in hindsight, it's clear that it was a choice with some amount of confusion potential for which we're sorry).
```
Comment (tianon) [inline README.md:50]: I think this blurb should probably move into our FAQ somewhere (since the name definitely generates questions on the regular).

**Review by tianon — COMMENTED** [review]
Nice, I feel like this is a much more straightforward introduction to the project.

(Just a few minor suggestions -- happy to discuss on them further though, and would like @yosifkit to weigh in too)

File: README.md (hunk 6 above)
Comment (wglambert) [reply README.md:50]: https://github.com/docker-library/faq/pull/19

File: README.md [hunk 7]
```diff
@@ -37,17 +37,21 @@
...
-See [Docker's documentation](https://docs.docker.com/docker-hub/official_repos/) for a good high-level overview of the program.
+The Docker Official Images are curated images [hosted on Docker Hub](https://hub.docker.com/u/library). The main tenets are:
+
+- Support [multiple architectures](#architectures-other-than-amd64)
 
-## What do you mean by "Official"?
+- Exemplify [`Dockerfile` best practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)
 
-In many cases, the images in this program are not only supported but also *maintained* directly by the relevant upstream projects.
+- [Actively rebuilt](#library-definition-files) for updates and security fixes
```
Comment (yosifkit) [inline README.md:48]:
```suggestion
- [Actively rebuild](#library-definition-files) for updates and security fixes
```

Something felt off on this point; it turns out that the tense was different from the rest of the list.

File: README.md (hunk 5 above)
Comment (yosifkit) [reply README.md:41]: Seems like a fine one to add, but should it be first? (I guess the real question, how are they ordered?)

**Review by yosifkit — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (tianon) [reply README.md:41]: I think the order is somewhat random ATM (maybe @wglambert can confirm or deny that).

IMO we should sort them in order of our own importance/consideration, but I think the order they're in now is probably good enough to start with?

Comment (wglambert) [reply README.md:41]: They're not in any particular order

**Pull request merged**